tests/test_contrib_loader.py

```diff
@@ -579,5 +579,44 @@ def test_replace_css_re(self):
         self.assertEqual(l.get_output_value('url'), [u'scrapy.org'])
 
 
+class SelectJmesTestCase(unittest.TestCase):
+        test_list_equals = {
+            'simple': ('foo.bar', '{"foo": {"bar": "baz"}}', "baz"),
+            'invalid': ('foo.bar.baz', '{"foo": {"bar": "baz"}}', None),
+            'top_level': ('foo', '{"foo": {"bar": "baz"}}', {"bar": "baz"}),
+            'double_vs_single_quoteString': ('foo.bar', '{"foo":{"bar":"baz"}}', "baz"),
+        }
+
+        def test_json_data(self):
+            for l in self.test_list_equals:
+                expr, test_list, expected = self.test_list_equals[l]
+                proc = Compose(json.loads, SelectJmes(expr))
+                test = proc(test_list)
+                self.assertEqual(
+                    test,
+                    expected,
+                    msg='test "{}" got {} expected {}'.format(l, test, expected)
+                )
+
+        def test_dict(self):
```

Comment:
Yes, use python types and forget about json, since we don't load json anymore inside the class. Also use the `test_list_equals` method, so it is easy to add more comparisons if needed.